docs/config_entries_diagnostics.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Config entry diagnostics
```

Comment:
Hmm, I personally think that it is not specific for a config_entry, rather for an integration

Reply:
Moved out of config entry and changed the title